ImagePicker/CameraCollectionViewCell.swift

```diff
@@ -44,15 +43,14 @@ open class CameraCollectionViewCell : UICollectionViewCell {
         view.contentMode = .scaleAspectFill
         return view
     }()
-    
-    var blurView: UIVisualEffectView?
-    
+    lazy var blurView: UIVisualEffectView = {
+        let result = UIVisualEffectView(effect: UIBlurEffect(style: .light))
+        return result
+    }()
```

Comment:
not sure if this is good idea, it will allocate blurView even when it's not used b/c it's used in layout subviews.

Reply:
Thanks, updated